static/app/gettingStartedDocs/javascript/nuxt.tsx

```diff
@@ -0,0 +1,374 @@
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ["localhost", /^https:\\/\\/yourserver\\.io\\/api/],`
+      : ''
+  }${
+    params.isReplaySelected
+      ? `
+  // Session Replay
+  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+  replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.`
+      : ''
+  }${
+    params.isProfilingSelected
+      ? `
+  // Set profilesSampleRate to 1.0 to profile every transaction.
+  // Since profilesSampleRate is relative to tracesSampleRate,
+  // the final profiling rate can be computed as tracesSampleRate * profilesSampleRate
+  // For example, a tracesSampleRate of 0.5 and profilesSampleRate of 0.5 would
+  // results in 25% of transactions being profiled (0.5*0.5=0.25)
+  profilesSampleRate: 1.0,`
+      : ''
+  }
+});
+`;
+
+const getSdkServerSetupSnippet = (params: Params) => `
+import * as Sentry from "@sentry/nuxt";
+
+Sentry.init({
+  dsn: "${params.dsn.public}",${
+    params.isPerformanceSelected
+      ? `
+  // Tracing
+  // We recommend adjusting this value in production, or using a tracesSampler for finer control.
+  tracesSampleRate: 1.0, // Capture 100% of the transactions
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ["localhost", /^https:\\/\\/yourserver\\.io\\/api/],`
+      : ''
+  }${
+    params.isProfilingSelected
+      ? `
+    // Set profilesSampleRate to 1.0 to profile every transaction.
+    // Since profilesSampleRate is relative to tracesSampleRate,
+    // the final profiling rate can be computed as tracesSampleRate * profilesSampleRate
+    // For example, a tracesSampleRate of 0.5 and profilesSampleRate of 0.5 would
+    // results in 25% of transactions being profiled (0.5*0.5=0.25)
+    profilesSampleRate: 1.0,`
+      : ''
+  }
+});
+`;
+
+const getVerifyNuxtSnippet = () => `
+<script setup>
+  const triggerError = () => {
+    throw new Error("Nuxt Button Error");
+  };
+</script>
+
+<template>
+  <button id="errorBtn" @click="triggerError">Trigger Error</button>
+</template>`;
+
+const getInstallConfig = () => [
+  {
+    language: 'bash',
+    code: [
+      {
+        label: 'npm',
+        value: 'npm',
+        language: 'bash',
+        code: 'npm install --save @sentry/nuxt',
+      },
+      {
+        label: 'yarn',
+        value: 'yarn',
+        language: 'bash',
+        code: 'yarn add @sentry/nuxt',
+      },
+      {
+        label: 'pnpm',
+        value: 'pnpm',
+        language: 'bash',
+        code: `pnpm add @sentry/nuxt`,
+      },
+    ],
+  },
+];
+
+const onboarding: OnboardingConfig = {
+  introduction: MaybeBrowserProfilingBetaWarning,
```

Comment:
one last thing. don't we want to add this here? 
```suggestion
 introduction: params => (
    <Fragment>
      <MaybeBrowserProfilingBetaWarning {...params} />
      <p>
        {tct('In this quick guide you’ll use [strong:npm], [strong:yarn] or [strong:pnpm] to set up:', {
          strong: <strong />,
        })}
      </p>
    </Fragment>
  ),
```